nextflow_schema.json

```diff
@@ -81,7 +81,8 @@
                 },
                 "orf_caller": {
                     "type": "string",
-                    "description": "Specify which ORF caller you would like to run."
+                    "description": "Specify which ORF caller you would like to run.",
```

Comment:
Add a description of possible alternatives, e.g. ", possible alternatives: prodigal, default: prodigal.". In the near future the list will be added to.
We might also want a fuller description, which goes after "help_text" IIRC.

Reply:
I changed it!